chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
I'm thinking about adding another selector to this line:
https://github.com/reepush/mooltipass/blob/1f1fc4ac00e2e86e942d876b3682aabec846ddf4/chrome_extension/vendor/mooltipass/mcCombinations.js#L1004

But it executes as a last case, when we don't have form element at all. We will submit form anyway and it won't help here. Not sure if we can check whether there are handlers on form submit or not and decide should we trigger submit.